.github/workflows/ci-test.yml

```diff
@@ -35,3 +32,21 @@ jobs:
       if: always()
       run: docker-compose -f dist/ci/docker-compose.yml down
 
+  linters:
+    runs-on: ubuntu-latest
+
```

Comment:
that's a a lot of copy&paste for 20s (from >20min). I just checked and looks like github doesn't support yaml anchors.

Reply:
yep. As it is just single docker image it can maybe use container directly, but it is still something we are checking...another idea is to add there also `perl -Wc` call for linters, but it will probably need adaptation of docker image to contain also required perl dependencies.